packages/@okta/vuepress-site/guides/sign-into-spa/section12/react/samples.md

```diff
@@ -0,0 +1 @@
+We have multiple samples available on our [React Samples](https://github.com/okta/samples-js-react) repo.
```

Comment:
We have multiple samples available in our [React Samples](https://github.com/okta/samples-js-react) repo.